### ISSUE TITLE
[5b3] setting Pin.Scale triggers sequence contains no element

### ISSUE THREAD

**Issue by softlion** [Mapsui/Mapsui#2765]
**Mapsui Version**
5.0.0-beta3

**Mapsui Platform**
Android

**Device**
Android simulator and devices

**Describe the bug**
When creating a new Pin, setting its Scale value throws an exception "sequence contains no element".

```
System.InvalidOperationException: Sequence contains no elements
   at System.Linq.ThrowHelper.ThrowNoElementsException()
   at System.Linq.Enumerable.First[IStyle](IEnumerable`1 source)
   at Mapsui.UI.Maui.Pin.OnPropertyChanged(String propertyName)
   at Mapsui.UI.Maui.Pin.set_Scale(Single value)
```

**To Reproduce**
Use that code to trigger the issue:
```c#
            var pin = new Pin
            {
                Tag = new(),
                Type = PinType.Pin,
                Position = new (0,0),
                Label = "label",
                Address = "address",
                Svg = "svg string",
                Color = Colors.Blue.ToMapsui(),
                Scale = .7f,
                Transparency = .45f, 
            };
```
Also, could you replace Transparency with the standard Opacity and reverse the values (1-n) ?

**Expected behavior**
regression: that does not crash in 4.x

It looks like it's trying to get a Style, which is not set.
```c#
((SymbolStyle) this.Feature.Styles.First<IStyle>()).SymbolScale = (double) this.Scale;
```

Also, that code does not work when a `VectorStyle` is added to Features first. It supports only a `LabelStyle`.

That also applies to `Pin.Transparency`. When a VectorStyle is set, you can not set `Pin.Transparency` (crash).

```c#
            var pin = new Pin
            {
                Tag = new(),
                Type = PinType.Pin,
                Position = new (0,0),
                Label = "label",
                Address = "address",
                Svg = "svg string",
                Color = Colors.Blue.ToMapsui(),
                Feature =
                {
                    // Styles = { new SymbolStyle { SymbolScale = .7f } }
                    Styles = { new VectorStyle
                    {
                        Fill = new Brush { Color = Colors.Blue.ToMapsui(), FillStyle = FillStyle.Svg, },
                        Outline = new Pen(Colors.Blue.ToMapsui()) { Width = 1, PenStyle = PenStyle.Solid },
                        //Opacity = 0.55f
                    } },
                },
                //Scale = .7f,
                Transparency = .45f, //this value is reversed. 0 = opaque. 1 = transparent.
            };
```

This also applies to `Pin.Anchor` which crashes if the first Style is not a SymbolStyle.

Finally, I see debug text on the map when interacting with the controls.  Not sure how to disable it. That was not appearing with 4.x

![image](https://github.com/user-attachments/assets/f61031e7-6c76-4172-a61b-dc464228ad1a)

**Comment by janusw** [comment]
I also see this. Looks like beta 2 is affected as well.

The problem can be reproduced via the `Mapsui.Samples.Maui.MapView` app: Run it, select the category `MapView` and then `Add many Pins Sample`. A red message appears instantly: `Error   Sequence contains no elements`

**Comment by janusw** [comment]
Probably closely related to #2764?

**Comment by janusw** [comment]
A quick bisection with the sample app between beta.1 and beta.2 points me towards PR #2632 as the culprit. In particular the error seems to be introduced by commit 8127f26afefbfb34b7e55d8d773bbc601adf71a0 by @pauldendulk. Paul, could you possibly have a look at this regression?

**Comment by inforithmics** [comment]
@janusw I'm currently looking at this, and yes my guess was to that this refactoring could cause this.

**Comment by pauldendulk** [comment]
Fixed with this PR: https://github.com/Mapsui/Mapsui/pull/2768

**Comment by janusw** [comment]
> Fixed with this PR: [#2768](https://github.com/Mapsui/Mapsui/pull/2768)

... in 5.0.0-beta.5 and upwards.
